README.md

```diff
@@ -1,239 +1,126 @@
 
-Interactive documentation of the API is provided using Swagger UI.
+## Ego - Secure Authentication & Authorization
 
-When run locally this can be found at: [http://localhost:8081/swagger-ui.html](http://localhost:8081/swagger-ui.html)
+<p align="center"><img alt="Ego in action" src="https://www.overture.bio/static/screenshot-21fc2cfc0ac1c3fd9bd7e62196477554.png"></p>
 
-<p align="center">
-    <img alt="arch" title="EGO Architecture" src="/docs/ego-arch.png" width="480">
-</p>
-<p align="center">
-  EGO Architecture
-</p>
+Access to sensitive and valuable information necessitates complex and secure methods to verify users and authorize what data and applications they are allowed to access. [Ego](https://www.overture.bio/products/ego/)  simplifies user management by providing a secure system to authenticate and authorize users of your application. [Ego](https://www.overture.bio/products/ego/) uses well-known single-sign-on identity providers like Google, GitHub, LinkedIn and ORCiD in place of managing usernames and passwords. 
 
-## Features
+Ego is [OAuth 2.0](https://oauth.net/2/) and [OpenID Connect](https://auth0.com/docs/authenticate/protocols/openid-connect-protocol) compliant. It is written in JAVA and uses [Sprint Boot](https://spring.io/projects/spring-boot) and [Spring Security Frameworks](https://spring.io/projects/spring-security). Because it uses [JSON Web Tokens (JWT)](https://jwt.io/) for stateless authorization, it can scale to accommodate many users. See our thorough [documentation](https://www.overture.bio/documentation/ego/) for further details. 
 
-Here are some of the features of EGO:
+## Related Products 
 
-- Single Sign on for microservices
-- User-authentication through Federated Identities such as Google, Facebook, Github (Coming Soon), ORCID (Coming Soon)
-- Uses JWT(Json Web Tokens) for Authorization Tokens
-- Provides ability to create permission lists for users and/or groups on user-defined permission entities
-- Built using well established Frameworks - Spring Boot, Spring Security
+<p align="left"><img alt="Overture overview" src="https://www.overture.bio/static/124ca0fede460933c64fe4e50465b235/a6d66/system-diagram.png"></p>
 
-## Tech Stack
+Overtures' modular architecture allows you to utilize and mix any of our products to fulfill your individual needs. Our core technologies, however, can also work in concert as an end-to-end data management system (DMS) designed to satisfy the needs of modern large-scale genomic research. For more information on our DMS, please see our [DMS documentation](https://www.overture.bio/documentation/dms/).
```

Comment:
Overture's

Reply:
I would start with

> Ego is a central component in the Overture Data Management System. < link to DMS >

then continue as is you have (slight text modifications).

> Overture's modular architecture allows you to utilize and mix any of our products to fulfill your individual needs. Our core technologies can be used as standalone services, and can alsobe deployed in concert as an end-to-end data management system (DMS) designed to satisfy the needs of modern large-scale genomics research.

Reply:
Note: I'm pluralizing `genomics` by reflex, I'm not sure if `genomic research` or `genomics research` or both are correct

Reply:
Got it!